builds/e2e/templates/e2e-run.yaml

```diff
@@ -19,7 +19,7 @@ steps:
     }
   displayName: Run tests
   env:
-    E2E_DPS_GROUP_KEY: $(TestDpsGroupKeySymmetric)
+    E2E_DPS_GROUP_KEY: '$(TestDpsGroupKeySymmetric)'
```

Comment:
Is there any other variable that can potentially be a base64 that also require this change?